### DISCUSSION TITLE
Self Hosting

### DISCUSSION THREAD

**Discussion by NixFull** [supabase/supabase#1817] [Questions] [answered]
Hi 

Has anyone already succeeded in self hosting

**Comment by inian** [comment]
Are there any issues when self hosting @NixFull?

**Comment by awalias** [comment, marked as answer]
just pinning the self hosting guide here for anyone coming after: https://supabase.io/docs/guides/self-hosting

we have a couple of users self hosting in environments like gov cloud etc. let us know if you're facing any issues @NixFull 👍
